### PR TITLE
Add loading state for initial render

### PR DESCRIPTION
**Pull request by lukehedger**

### CODE REVIEW

File: app/core/components/index.js [hunk 1]
```diff
@@ -7,6 +9,8 @@ import MetaIdInput from './MetaIdInput'
 import Search from './Search'
 
 export {
+  AppLoader,
+  Fade,
```
Comment (MrTibbles) [inline app/core/components/index.js]: Potential additions to primitives library.

File: app/core/primitives/View/View.js [hunk 2]
```diff
@@ -9,6 +9,8 @@ import { View as ViewPrimitive } from 'jaak-primitives'
  * @return {Function} React component
  */
 const StyledView = styled(ViewPrimitive)`
+  background-color: ${({ backgroundColor, theme }) =>
```
Comment (MrTibbles) [inline app/core/primitives/View/View.js]: We need to revisit the primitives library, adding such params as this.

File: app/domains/ui/selectors.js [hunk 3]
```diff
@@ -20,6 +20,15 @@ const getError = createSelector([getAll], state => {
   return state.get('error') && state.get('error').toJS()
 })
 
+/**
+ * Get application initial load status
+ *
+ * @type {Boolean}
+ */
+const getIsInitialLoad = createSelector([getAll], state => {
```
Comment (MrTibbles) [inline app/domains/ui/selectors.js]: Singular injection doesnt require `[]`, but minor in the grand scheme of things.

File: app/core/primitives/View/View.js (hunk 2 above)
Comment (lukehedger) [reply app/core/primitives/View/View.js]: Yes 100%. This View should not be here really.

File: app/domains/ui/selectors.js (hunk 3 above)
Comment (lukehedger) [reply app/domains/ui/selectors.js]: Good spot

**Pull request merged**